resources/user-agents/crawlers/microsoft/bingpreview-beta.json

```diff
@@ -80,54 +91,59 @@
       },
       "children": [
         {
-          "match": "Mozilla/5.0*(iPad#PLATFORM#)*applewebkit*(*KHTML? like Gecko*)*Version/*Safari/* BingPreview/#MAJORVER#.#MINORVER#*",
-          "device": "iPad",
+          "match": "Mozilla/5.0 (#PLATFORM# Trident/7.* IEMobile/*) like Gecko BingPreview/#MAJORVER#.#MINORVER#*",
+          "engine": "Trident_7",
           "platforms": [
-            "iOS_A_3_0", "iOS_A_3_1", "iOS_A_3_2", "iOS_A_4_0", "iOS_A_4_1", "iOS_A_4_2", "iOS_A_4_3",
```

Comment:
I forgot to mention in the description that I removed all of these iOS versions because there's no evidence that these bots use these agents, so they just add unnecessary pattern bloat, IMO.

I looked through our logs and other parser test suites and didn't find any other cases of iOS other than 6.0 and 7.0 being used by the bing bots.

The `iOS_A` pattern should pick up any new versions that Microsoft decides to use (they just won't have the right iOS version).  Browscap's tests don't hit any of these versions either (obviously :) ).